README.md

```diff
@@ -18,9 +18,12 @@ More details: http://pjreddie.com/darknet/yolo/
 
 0.  [Improvements in this repository](#improvements-in-this-repository)
 1.  [How to use](#how-to-use-on-the-command-line)
-2.  [How to compile on Linux](#how-to-compile-on-linux)
+2.  How to compile on Linux
+    * [Using cmake](#how-to-compile-on-linux-using-cmake)
+    * [Using make](#how-to-compile-on-linux-using-make)
 3.  How to compile on Windows
     * [Using vcpkg](#how-to-compile-on-windows-using-vcpkg)
+    * [Using Cmake-GUI](#how-to-compile-on-windows-using-cmake-gui)
```

Comment:
I think `Using Cmake-GUI` should be the first, because it is shorter.
And `Using vcpkg` should be the second

Reply:
Noted and resolved in latest commit.